### PR TITLE
Cloning data with activities

### PR DESCRIPTION
**Pull request by ajainuary**
**Description**
This PR modifies the cloning logic for activities in the wiki to allow cloning of the data with the activities

**Resolved Issues**
This is a follow up from #1723

### CODE REVIEW

**Comment by houshuang** [comment]
@ajainuary Can you fix merge conflict please?

File: frog/imports/api/generateReactiveFn.js [hunk 1]
```diff
@@ -189,10 +189,11 @@ export class Doc {
     );
     if (LIData.liType === 'li-activity') {
       // In this case the activity also needs to be duplicated
+      const ac = connection.get('rz', LIData.payload.rz);
+      ac.fetch();
```
Comment (houshuang) [inline frog/imports/api/generateReactiveFn.js]: This is asynchronous, so you will probably have timing issues here in production (not on localhost). You should use a callback or Promise). See other places where we use connection.get - we always wait for onload etc.

File: frog/imports/client/Wiki/WikiTopNavbar.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
+import React from 'react';
```
Comment (houshuang) [inline frog/imports/client/Wiki/WikiTopNavbar.js]: What happened to this file - is this because of a merge conflict?

**Review by houshuang — CHANGES_REQUESTED** [review]

File: frog/imports/client/Wiki/WikiTopNavbar.js (hunk 2 above)
Comment (ajainuary) [reply frog/imports/client/Wiki/WikiTopNavbar.js]: Yes, somehow git merge re-introduced this file

Comment (ajainuary) [reply frog/imports/client/Wiki/WikiTopNavbar.js]: (Removed now)

File: frog/imports/api/generateReactiveFn.js (hunk 1 above)
Comment (ajainuary) [reply frog/imports/api/generateReactiveFn.js]: Now using await function

**Review by houshuang — APPROVED** [review]

**Pull request merged**